.github/workflows/master_build.yml

```diff
@@ -0,0 +1,114 @@
+name: Master CI Build
+
+on:
+  push:
+    branches:
+#      - master
+      - translation-experiment
+
+jobs:
+  build_en_docs:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    name: Build and deploy english docs
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -U Sphinx==2.4.4
+        pip install --upgrade recommonmark
+        pip install sphinx_rtd_theme
+    - name: Build documentation
+      shell: bash
+      run: |
+        cd docs
+        make html
+        mkdir -p build-archives
+        cd build/html/
+        tar -czvf ../../build-archives/build-en.tar.gz *
+    - name: Upload documentation
+      uses: JulianGro/SFTP-Deploy-Action@v1.4
+      with:
+        username: 'gha'
+        server: '94.130.177.235'
```

Comment:
I see this ip address repeated over the entire file, perhaps consider setting it as a variable that can be re-used through the script. same for user name.  In case they change you'd only have to change those it will only be in one place.

Reply:
From what I can see online, there isn't actually normal variables.
I could set environment variables, but those do not persist between runners, which is pretty much the whole point.